ubuntu/check_distro_for_ubuntu.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+# check distribution for Ubuntu
+
+check_distro_for_ubuntu() {
+    echo "Started checking distribution for Ubuntu at $(date)"
+
+    grep -i ubuntu /etc/*-release &>/dev/null
+    if [[ $? == 0 ]]; then 
+        echo "Distribution is Ubuntu."
+        cat /etc/os-release
+        echo ""
+
+        echo "Finished checking distribution for Ubuntu at $(date)"
+        echo ""
```

Comment:
@VicPhanDevOps , why the extra echo statement here and on ln 23?
why not end the previous echo with a `\n`?

Reply:
I like to use an extra `echo ""` statement for blank spaces if the previous `echo` statement ends with any special characters.  I don't want sequential special characters to be hard for people to read (Example: `)\n"`).